ThirdParty/Bolts/BoltsInClasspath/BoltsInClasspath/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  BoltsInClasspath
+//
+//  Created by siva kongara on 1/14/17.
+//  Copyright © 2017 venkat kongaraSiva. All rights reserved.
```

Comment:
>2017 venkat kongaraSiva [](start = 16, length = 23)

Venkat, you'll want to use the same Copyright header as the rest of the code in -samples, updated for 2017

(sample)
[https://github.com/Microsoft/WinObjC-Samples/blob/master/Scenarios/Maps/Maps/MapsSample/AppDelegate.h](https://github.com/Microsoft/WinObjC-Samples/blob/master/Scenarios/Maps/Maps/MapsSample/AppDelegate.h)

//******************************************************************************
//
// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
//
// This code is licensed under the MIT License (MIT).
//
// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
// THE SOFTWARE.
//
//******************************************************************************